frontend/src/navigation/router.tsx

```diff
@@ -0,0 +1,14 @@
+import {Route, Router, Routes} from "react-router-dom";
+import {MainPage} from "../pages/MainPage.tsx";
+import {LogInPage} from "../pages/LoginPage.tsx";
```

Comment:
Wrong import. There is no LogInPage in "../pages/LoginPage.tsx", only LoginPage.
This file is also unused.